lib/debugger/hookWorld.luau

```diff
@@ -1,14 +1,37 @@
+--#selene:allow(empty_loop)
 local useCurrentSystem = require(script.Parent.Parent.topoRuntime).useCurrentSystem
 local World = require(script.Parent.Parent.World)
+local rollingAverage = require(script.Parent.Parent.rollingAverage)
 
 local originalQuery = World.query
 
 local function hookWorld(debugger)
 	World.query = function(world, ...)
 		if useCurrentSystem() == debugger.debugSystem then
+			local start = os.clock()
+
+			for _ in originalQuery(world, ...) do
+			end
```

Comment:
Is this intentional?

Reply:
Yep, it duplicates the query as to not drain the original one and iterates it